plugins/nf-slack/src/test/nextflow/slack/SlackDslTest.groovy

```diff
@@ -0,0 +1,85 @@
+package nextflow.slack
+
+import nextflow.plugin.Plugins
+import nextflow.plugin.TestPluginDescriptorFinder
+import nextflow.plugin.TestPluginManager
+import nextflow.plugin.extension.PluginExtensionProvider
+import org.pf4j.PluginDescriptorFinder
+import spock.lang.Requires
+import spock.lang.Shared
+import spock.lang.Timeout
+import test.Dsl2Spec
+import test.MockScriptRunner
+
+import java.nio.file.Path
+
+
+/**
+ * @author : jorge <jorge.aguilera@seqera.io>
+ *
+ */
+@Timeout(10)
+class SlackDslTest  extends Dsl2Spec{
+
+    @Shared String pluginsMode
+
+    def setup() {
+        // reset previous instances
+        PluginExtensionProvider.reset()
+        // this need to be set *before* the plugin manager class is created
+        pluginsMode = System.getProperty('pf4j.mode')
+        System.setProperty('pf4j.mode', 'dev')
+        // the plugin root should
```

Comment:
Part of the comment seems to be missing or at least I can't guess right now what the root should.

Reply:
hahaha, I think it was missing from the origin:

https://github.com/nextflow-io/nf-hello/blob/397b06581fd1b1f7d0e9c488897088bd4de860d5/plugins/nf-hello/src/test/nextflow/hello/HelloDslTest.groovy#L31-L31